website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -4,36 +4,38 @@ id: "user-tokens"
+:::
 
-The current API key is located under **Personal Settings → API Key.** 
+Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using service Tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
 
-Please [contact support](/docs/dbt-support#dbt-cloud-support) with any questions or concerns.
+PATs inherit the permissions of the user that created them. For example, if a developer-licensed user with Project Admin role access to specific projects creates a PAT, the token will get the Project Admin role with access to the same projects as the user. These tokens are also account-specific, so if a user has access to more than one dbt Cloud account with the same email address, they need to create a unique PAT for each one of these accounts. 
 
-:::
+## Create a personal access token
 
-## Account-scoped personal access tokens
+Creating an account scoped PAT requires only a few steps.
+1. Navigate to your **Account Settings**, expand **API tokens** and click **Personal tokens**.
+2. Click **Create personal access token**.
+3. Give the token a descriptive name and click **Save**. 
 
-:::info New 
+Copy the token prior to closing the window. _It will not be available after and you will have to create a new token if you lose it._ 
 
-On Feb 7, 2024, we introduced a new type of token for individual users called personal access tokens. Note that these differ from [Service Tokens or API Keys](/docs/dbt-cloud-apis/authentication#types-of-api-access-tokens). Before this release, user API keys were the only way to access dbt Cloud API on behalf of the user. These API Keys were user-specific and were not scoped to an account. To enhance the security of dbt Cloud, we are moving away from this model to account-specific tokens. 
+### Rotate your tokens
 
-:::
+It's recommended that you rotate your PATs at regular intervals for security purposes. To rotate your PATs, create a new token and, once it's in place, delete the old token.
 
-Each dbt Cloud user with a [Developer license](https://docs.getdbt.com/docs/cloud/manage-access/seats-and-users) can create a new personal access token (PAT) to access the dbt Cloud API and dbt Cloud CLI. This token can execute queries against the dbt Cloud API on the user's behalf. To access dbt Cloud APIs and resources on behalf of the _account_, we recommend using service Tokens instead. Learn more about [which token type you should use](/docs/dbt-cloud-apis/authentication#which-token-type-should-you-use) to understand the token differences.
+To delete a token:
```

Comment:
This might be easier for folks to find if we elevate it to an L2. Whatever you decide for creating a PAT, you should do here identifying the acronym in the header or 1st paragraph.

```suggestion
## Delete a personal access token (PAT)

If you're rotating your tokens, make sure to create a new token before deleting the old one.

To delete a PAT:
```

Reply:
We define (PAT) in the headers above, it might be a little overkill here. Makes the link simpler to leave it off